hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/dn/scanner/TestBackgroundContainerDataScannerIntegration.java

```diff
@@ -85,7 +85,7 @@ void testCorruptionDetected(TestContainerCorruptions corruption)
     assertTrue(containerChecksumFileExists(containerID));
 
     waitForScmToSeeReplicaState(containerID, CLOSED);
-    long initialReportedDataChecksum = getContainerReplica(containerID).getDataChecksum();
+    long initialReportedDataChecksum = getContainerReplica(containerID).getChecksums().getDataChecksum();
```

Comment:
For this use case we only need to use the `ContainerChecksums` objects and compare them. We don't need to further extract the long values.

There is a new `assertReplicaChecksumMatches` introduced in #8565 that can take the `ContainerChecksums` object as a parameter instead of a long.